pkg/minikube/command/command_runner.go

```diff
@@ -84,3 +85,101 @@ func (rr RunResult) Output() string {
 	}
 	return sb.String()
 }
+
+// teePrefix copies bytes from a reader to writer, logging each new line.
+func teePrefix(prefix string, r io.Reader, w io.Writer, logger func(format string, args ...interface{})) error {
+	scanner := bufio.NewScanner(r)
+	scanner.Split(bufio.ScanBytes)
+	var line bytes.Buffer
+
+	for scanner.Scan() {
+		b := scanner.Bytes()
+		if _, err := w.Write(b); err != nil {
+			return err
+		}
+		if bytes.IndexAny(b, "\r\n") == 0 {
+			if line.Len() > 0 {
+				logger("%s%s", prefix, line.String())
```

Comment:
does this mean, for binary files or big files this will flood the logs?

Reply:
This function is only used for streaming SSH commands - and remains unchanged by this PR. The only thing I did was move which file it was located in.